app/assets/stylesheets/modules/digital.scss

```diff
@@ -1,42 +1,163 @@
-
 /* Digital pack */
-.digital-cta {
-    padding-top: $gs-baseline*2;
-    padding-bottom: $gs-baseline*3;
+
+.digital-pack {
+    .page-header {
+        margin-bottom: $gs-baseline*2;
+    }
+}
+
+// CONTAINERS
+
+$left-column: gs-span(2);
+
+.digital-pack {
+    .gs-container {
```

Comment:
Is there a simpler way to think about this? Modifying the global `gs-container` class based on a body class feels a little awkward.